src/PhpGenerator/ClassType.php

```diff
@@ -25,7 +25,7 @@ class ClassType extends Nette\Object
 	/** @var PhpNamespace|NULL */
 	private $namespace;
 
-	/** @var string */
+	/** @var string|NULL */
 	private $name;
```

Comment:
`Parameter` & `Property` uses `$name = ''` instead of `|NULL`, because `$name` is not "optional" parameter.